pkg/controller/operands/kubevirt.go

```diff
@@ -461,6 +461,9 @@ func getKVDevConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.DeveloperCon
 	if useKVMEmulation {
 		devConf.UseEmulation = useKVMEmulation
 	}
+	if lv := hc.Spec.LogVerbosityConfig; lv != nil && lv.Kubevirt != nil {
```

Comment:
I'm afraid this is not a complete reconciliation. 

We only check that HC field is not nil. But what if it is and the KV is not? In this case we won't force KV field to be nil as well, and then we're entering an endless reconcile loop, where we do find a gap between KV and HCO, but fail to close this gap.

Reply:
I was wrong. This code will work fine. The comparison is done later.